lucille-core/src/main/java/com/kmwllc/lucille/util/JSONUtils.java

```diff
@@ -0,0 +1,112 @@
+package com.kmwllc.lucille.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.io.IOException;
+
+public class JSONUtils {
```

Comment:
took another look at this util class and thinking maybe we don't need it... instead, we could put a new method on the Document interface like this:

`  default JsonNode getNestedJson(String name) {
    String[] pathSegments = name.split("\\.");

    if (pathSegments.length == 0 || !has(pathSegments[0])) {
      return null;
    }
    JsonNode node = getJson(pathSegments[0]);

    for (int i = 1; i < pathSegments.length; i++) {
      String segment = pathSegments[i];

      if (!node.has(segment)) {
        return null;
      }

      node = node.get(segment);
    }

    return node;
  };`

along with a similar Document.setNestedJson method...